docs/src/environments/_index.md

```diff
@@ -54,106 +89,91 @@ Each new environment you create is considered a **child** of the **parent** envi
 
 Each child environment can [sync](../other/glossary.md#sync) code and/or data down from its parent
 and [merge](../other/glossary.md#merge) code up to its parent.
-The child environments are used for development, staging, and testing.
+You can use child environments for development, staging, and testing.
 
 When you [branch](../other/glossary.md#branch) an environment to create a new child environment,
 its parent is the environment it was created from.
 If you push a branch through Git or a [source integration](../integrations/source/_index.md),
-the parent defaults to the default environment.
+the parent environment is your [default environment](#default-environment).
```

Comment:
:lady_beetle: Might be worth mentioning what I think the original was trying to get at (but not really succeeding): that you can set different parents through the CLI when branching/pushing.